packages/frontend2/src/app/_components/projects/sections/chart-section.tsx

```diff
@@ -11,9 +12,18 @@ type ChartSectionId = Extract<
   'onchain-costs' | 'tvl' | 'activity'
 >
 
+/* 
+  Note (torztomasz):
+  I know this props with optional tokens and stacked is kinda bad as it is only used for TVL chart.
+  Although I couldn't get type union working with get-l2-project-details.tsx or get-l3-project-details.tsx.
+  You can give it a shot if you want.
+  Shots counter: 0
+*/
```

Comment:
Add a task in Linear, I'll tackle it when I'll be looking for a challenge :eyes:

Reply:
let's keep it lowkey without linear 😅